RecoMuon/SimMuonAlgos/plugins/MuonSimClassifier.cc

```diff
@@ -0,0 +1,574 @@
+// -*- C++ -*-
+//
+// Package:    MuonSimClassifier
+// Class:      MuonSimClassifier
+//
+/**\class MuonSimClassifier MuonSimClassifier.cc MuonAnalysis/MuonAssociators/src/MuonSimClassifier.cc
```

Comment:
this is out of sync

Reply:
After reviewing the implementation of this producer, it looks like it can simply be placed in SimMuon/MCTruth,
which already hosts most of the tools needed and has the necessary dependencies.

Earlier, when I proposed  RecoMuon/SimMuonAlgos, I thought that the final producer implementation will be creating reco::Muons or something from DataFormats/MuonReco

Reply:
@davidlange6 
please confirm about moving to SimMuon/MCTruth.
I can see a valid argument in association to a category by the target application configuration and the target data tier (this will be essentially a part of the "RECO" processing step).
Still, essentially all of the code here is a reduction of data made by muonAssociatorByHits and doesn't have much to do with reco algorithms.

Reply:
I personally don't care where it goes. Just decide and tell me.

Reply:
Moved to SimMuon/MCTruth.